### ISSUE TITLE
When will Fabric 1.17 version be available for download?

### ISSUE THREAD

**Issue by kristibektashi** [Barteks2x/CubicChunks#644]
# Version Information
Minecraft version:  21w11a
Forge version:  None
Other mods (if applicable):  

# Steps to reproduce:
1. 
2. 
3. 

Expected Behavior:

Actual Behavior:

Link to crash log (if applicable - use a site such as [hastebin](https://hastebin.com/):

**Comment by AsheX1** [comment]
build it ;)

**Comment by CorgiTaco** [comment]
***TODO TO GET 1.17 INTO A FUNCTIONAL AND A "WORKING" STATE:***

-Light Engine
-Fluid & Block Ticks
-World gen(Structures and ALL Range decorated features like ores)
-Entity tracker + Storage(Saving)
-Height map saving
-Undo sacrificing x/z space and do NOT pack blockpos to long in places such as the light engine.
-3D biome features

***1.17 INITIAL ALPHA Release:***
-(All previously mentioned)
-Vanilla worlds -> Cubic Worlds
-Performance optimizations
-Bug fixes/Stability
-Take control of fabric api where required.
-Hybrid Worlds

***1.17 ALPHA Releases***
-Stability and bug fixing across the board
-Expose different mod compatibilities via datapacks (i.e structure vertical spacing/seperation distances that structure mods can argue and other powerful configurations for other types of mods) this allows for mods to not have a required HARD dependency on Cubic Chunks.
-Mod compatibility
-Performance Optimizations

***1.17 BETA Releases***
-Relatively low known issues/little maintenance

***1.17 RELEASES***
-Good luck seeing this in your lifetime

So basically... not any time soon

**Comment by kristibektashi** [comment]
Well rip
